Gaming The Syllabus/Assets/Scripts/Combat/Action Effects/RandomHealTarget.cs

```diff
@@ -0,0 +1,58 @@
+using System.Runtime.Versioning;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Xml.Serialization;
+using System.Xml.Schema;
+using System.Xml;
+
+public class RandomHealTarget : ActionEffect
+{
+    public int minHeal { get; private set; }
+    public int maxHeal { get; private set; }
+
+    public RandomHealTarget(int minHeal, int maxHeal, TargetType target)
+    {
+        this.minHeal = minHeal;
+        this.maxHeal = maxHeal;
+        Target = target;
+    }
+
+    public RandomHealTarget()
```

Comment:
Annoys me to no end that we have to have this no-args constructor around for XML Serialization... wonder if there is a way around that